src/services/FlawService.ts

```diff
@@ -8,49 +8,17 @@ import {
   type OsidbFetchOptions
 } from '@/services/OsidbAuthService';
 import { createCatchHandler, createSuccessHandler } from '@/composables/service-helpers';
-import { isFlawIdentifierValid } from '@/utils/helpers';
-
-import { ZodFlawSchema } from '@/types/zodFlaw';
-
-
-function isFlaw(data: any) {
-  return !!ZodFlawSchema.safeParse(data).success;
-}
-
-function parseFlawId(options: any) {
-  if (isFlaw(options.data) && options.data.uuid) {
-    return options.data.uuid;
-  }
-
-  // Look for the presence of a flaw id on Affects, CVSS Scores, and other entities 'belonging to' flaws
-  if (options.data?.flaw && isFlawIdentifierValid(options.data.flaw?.toString())) {
-    return options.data.flaw;
-  }
-
-  const flawOsidbIdRegex = /flaws\/(.+?)\/.*/;
-  const maybeFlawId = options.url.match(flawOsidbIdRegex)?.[1];
-  if (maybeFlawId) {
-    return maybeFlawId;
-  }
-  return null;
-}
 
 export async function beforeFetch(options: OsidbFetchOptions) {
   if (options.data && ['PUT', 'POST'].includes(options.method.toUpperCase())) {
-    const flawId = parseFlawId(options);
-    if (flawId === null) {
-      return;
-    }
     try {
-      const flaw = await getFlawUpdatedDt(flawId);
-      options.data.updated_dt = flaw.updated_dt;
```

Comment:
One last question. When this got deleted, how does the `updated_dt` get sent in the request body?

Reply:
When that got deleted made the `updated_dt` being send in the request the same as before, which is wrong.
Indeed the just fetched `updated_dt` is the one that should go in the request.

Again, thanks for noting it, that was essential.